### PR TITLE
Fix transaction broadcast - Closes #3459, #3467

### LINKED ISSUE #3459: Adapt transaction broadcast
### Expected behavior
The client should be able to broadcast transactions to lisk service.

### Actual behavior
The client broadcasts the transactions to lisk core.

### PR DESCRIPTION
**Pull request by UsamaHameed**
### What was the problem?
This PR resolves #3459, #3467

### How was it solved?
- Fixes issues with transaction broadcast
- Added transformer function to convert the broadcasted transaction back to lisk-service format to ensure compat with UI
- Polyfilled native bigint with `toJSON` function for `JSON.stringify` to be able to serialize it.

Additionally: 
- Fixed a few issues where wallet/votes and wallet/delegateProfile views were crashing
- Fixed an issue with the action `votesRetrieved`

### CODE REVIEW

File: src/utils/account.js [hunk 1]
```diff
@@ -16,22 +16,36 @@ export const extractPublicKey = (passphrase) => {
   return undefined;
```
Comment (reyraa) [inline src/utils/account.js:16]: These retuened `undefined` are essentially silent failures. I think we should add a logger here so we don't overlook the broken features.

Comment (reyraa) [reply src/utils/account.js:16]: Same goes for the other two convertors.

File: src/utils/api/account/lsk.js [hunk 2]
```diff
@@ -69,6 +72,7 @@ export const getAccount = async ({
 }) => {
   const normParams = getAccountParams(params);
 
+  console.log({ params, normParams });
```
Comment (reyraa) [inline src/utils/api/account/lsk.js:75]: Remove?

File: src/utils/api/transaction/lsk.js [hunk 3]
```diff
@@ -17,6 +17,7 @@ import ws from '../ws';
 import { getDelegates } from '../delegate';
 import regex from '../../regex';
 import { validateAddress } from '../../validators';
+import { extractAddressFromPublicKey } from '../../account';
```
Comment (reyraa) [inline src/utils/api/transaction/lsk.js:20]: Should we use aliases here?

File: src/utils/api/transaction/lsk.js [hunk 4]
```diff
@@ -219,51 +220,106 @@ const splitModuleAndAssetIds = (moduleAssetId) => {
...
-    senderPublicKey: Buffer.from(senderPublicKey, 'hex'),
-    nonce: BigInt(nonce),
-    fee: BigInt(fee),
-    signatures: [],
+export const transformTransaction = (transaction) => {
+  const moduleAssetId = [transaction.moduleID, transaction.assetID].join(':');
+  const senderAddress = extractAddressFromPublicKey(transaction.senderPublicKey);
+  const senderPublicKey = transaction.senderPublicKey.toString('hex');
+
+  console.log(transaction);
```
Comment (reyraa) [inline src/utils/api/transaction/lsk.js:228]: Don't forget to remove this.

File: src/utils/api/transaction/lsk.js [hunk 5]
```diff
@@ -219,51 +220,106 @@ const splitModuleAndAssetIds = (moduleAssetId) => {
...
+  const transformedTransaction = {
+    id: transaction.id.toString('hex'),
+    moduleAssetId,
+    fee: String(transaction.fee),
+    nonce: String(transaction.nonce),
+    sender: { publicKey: senderPublicKey, address: senderAddress },
+    signatures: transaction.signatures,
   };
 
   if (moduleAssetId === MODULE_ASSETS_NAME_ID_MAP.transfer) {
```
Comment (reyraa) [inline src/utils/api/transaction/lsk.js:238]: A switch-case seems more readable here.

File: src/utils/api/transaction/lsk.js [hunk 6]
```diff
@@ -219,51 +220,106 @@ const splitModuleAndAssetIds = (moduleAssetId) => {
...
+    const transaction = {
+      moduleID,
+      assetID,
+      senderPublicKey: Buffer.from(senderPublicKey, 'hex'),
+      nonce: BigInt(nonce),
+      fee: BigInt(fee),
+      signatures: [],
+    };
+
+    if (moduleAssetId === MODULE_ASSETS_NAME_ID_MAP.transfer) {
```
Comment (reyraa) [inline src/utils/api/transaction/lsk.js:290]: A switch-case seems more readable here.

**Review by reyraa — CHANGES_REQUESTED** [review]

File: src/utils/api/transaction/lsk.js (hunk 3 above)
Comment (UsamaHameed) [reply src/utils/api/transaction/lsk.js:20]: Already done in the next PR.

File: src/utils/api/transaction/lsk.js (hunk 6 above)
Comment (UsamaHameed) [reply src/utils/api/transaction/lsk.js:290]: Taken care of in the next PR.

**Review by reyraa — APPROVED** [review]

**Pull request merged**